### PR TITLE
Update Getting Started topic and graphics

### LINKED ISSUE #9228: [Docs] Refactor content under the Get Started heading.

### PR DESCRIPTION
**Pull request by JStickler**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:
Final PR to complete fixes #9228 

**Special notes for your reviewer**:
Updates Getting Started topic to be a general overview, links to Grafana topics, and Agent configuration file.
Adds new Quick Start topic for running Loki Docker compose file locally that includes working sample queries.
Adds three new graphics
Removes two unused graphics

### CODE REVIEW

**Review by monodot — CHANGES_REQUESTED** [review]
I've made some suggestions based on running through the examples. I think we should update the metric query to ensure it draws a single metric line on a graph, and also update the Docker Compose setup so that Flog writes more lines per second, which will make the chart more meaningful and so easier to understand for first-timers.

**Review by monodot — APPROVED** [review]
Great, this looks good to me.

File: docs/sources/get-started/quick-start.md [hunk 1]
```diff
@@ -0,0 +1,172 @@
+---
+title: Quick start to run Loki locally
+menuTitle: Loki quick start
+weight: 550
+description: How to create and use a simple local Loki cluster for testing and evaluation purposes.
+---
+
+# Quick start to run Loki locally
+
+If you want to experiment with Loki, you can run Loki locally using the Docker Compose file that ships with Loki. It runs Loki in a [scalable monolithic deployment](https://grafana.com/docs/loki/latest/get-started/deployment-modes/#simple-scalable) mode and includes a sample application to generate logs.
```
Comment (MichelHollands) [inline docs/sources/get-started/quick-start.md:10]: Is that URL correct? Doesn't it point to the SSD version instead of the monolithic one?

**Review by MichelHollands — APPROVED** [review]
Nice. I left 1 question but looking fine otherwise.

**Pull request merged**